SDK_dotnet/Clients/GluwaClient.cs

```diff
@@ -304,7 +302,7 @@ public GluwaClient(Environment env)
             var result = new Result<bool, ErrorResponse>();
             var requestUri = $"{mEnv.BaseUrl}/v1/Transactions";
 
-            Result<FeeResponse, ErrorResponse> getFee = await getFeeAsync(currency);
+            Result<FeeResponse, ErrorResponse> getFee = await getFeeAsync(currency, amount);
```

Comment:
Will this work for all currencies? Only sUSDCG has the variable fee that requires amount to be provided

Reply:
Yes, Fee service will handle it.

Reply:
It would be cleaner to not send an amount when you don't need to @KyoungBaeLee